src/components/browse/components/ControlsAndResults.js

```diff
@@ -83,7 +83,6 @@ export class ControlsAndResults extends React.PureComponent {
             // From ColumnCombiner or CustomColumnController (if props.hideColumns present):
             columnDefinitions,
             // From SelectedItemsController:
-            onCompleteSelection, onCancelSelection,
```

Comment:
Instead of defining these funcs in SelectedItemsController and passing them down into SelectStickyFooter, which just define the functions in SelectStickyFooter instead (since all the data for them is available in `selectedItems`).